conf/application.conf

```diff
@@ -104,56 +114,76 @@ controllers {
 # You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
 
 # Root logger:
-logger.root=ERROR
+logger.root = ERROR
 
 # Logger used by the framework:
-logger.play=INFO
+logger.play = INFO
 
 # Logger provided to your application:
-logger.application=DEBUG
+logger.application = DEBUG
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-    name = ${appName}
-    rateUnit = SECONDS
-    durationUnit = SECONDS
-    showSamples = true
-    jvm = true
-    enabled = true
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
 }
 
 
+Test {
+  metrics {
+    enabled = false
```

Comment:
This do not work now -  Test.metrics.enabled is ignored, only Test.microservice.metrics.graphite.enabled is taken in account.